table_evaluator/table_evaluator.py

```diff
@@ -354,17 +372,28 @@ def score_estimators(self):
             raise Exception(f'self.target_type should be either \'class\' or \'regr\', but is {self.target_type}.')
         return results
 
-    def visual_evaluation(self, **kwargs):
+    def visual_evaluation(self, save_dir=None, **kwargs):
         """
         Plot all visual evaluation metrics. Includes plotting the mean and standard deviation, cumulative sums, correlation differences and the PCA transform.
-
+        :save_dir: directory path to save images 
         :param kwargs: any kwargs for matplotlib.
         """
-        self.plot_mean_std()
-        self.plot_cumsums()
-        self.plot_distributions()
-        self.plot_correlation_difference(**kwargs)
-        self.plot_pca()
+        if save_dir is None: 
+            self.plot_mean_std()
+            self.plot_cumsums()
+            self.plot_distributions()
+            self.plot_correlation_difference(**kwargs)
+            self.plot_pca()    
+        else: 
+            if not os.path.isdir(save_dir):
+                os.makedirs(save_dir)
+
+            self.plot_mean_std(fname=os.path.join(save_dir, 'mean_std.png'))
```

Comment:
Could you replace these `os.path.join` with 
```python
from pathlib import Path
save_dir = Path(save_dir)

self.plot_mean_std(fname=save_dir/'mean_std.png'
```

and the same for the other paths. I prefer working with the path library over os.path.join.